### ISSUE TITLE
Use `Path` type for paths, not `String`s?

### ISSUE THREAD

**Issue by nickrobinson251** [invenia/PkgTemplates.jl#100]
I noticed #61 still uses Strings for filepaths, but we could instead use https://github.com/rofinn/FilePathsBase.jl

**Comment by christopher-dG** [comment]
Yeah I thought about it very briefly but didn't look into it too hard. Provided that it doesn't affect the user API, I'm good with it.

**Comment by nickrobinson251** [comment]
agreed, we shouldn't affect public API; the suggestion is use it behind the scenes to hopefully make maintainence easier :)

**Comment by christopher-dG** [comment]
I am going to leave FilePaths out for now, it can almost definitely be added later though.
